ci/benchmarks/partial-conv/esm2_pretrain.yaml

```diff
@@ -0,0 +1,54 @@
+scope: partial-conv
```

Comment:
how do we assess convergence? Is there a script later that queries the wandb logs?

Reply:
I will add tests which check if final metrics are within some limits